tree/dataframe/src/RCsvDS.cxx

```diff
@@ -247,7 +299,11 @@ size_t RCsvDS::ParseValue(const std::string &line, std::vector<std::string> &col
       }
    }
 
+   if (prev_i == i) // empty entry not in the last column
+      val << fFillValue;
    columns.emplace_back(val.str());
+   if (i == line.size() - 1 && line[i] == fDelimiter) // empty entry in the last column
+      columns.emplace_back(fFillValue);
```

Comment:
maybe simpler:
```suggestion
   if (prev_i == i || (i == line.size() - 1 && line[i] == fDelimiter)) // empty cell
      columns.emplace_back(fFillValue);
   else
      columns.emplace_back(val.str());
```

Reply:
That would not work. Reason is that `ParseValue` is not called if the last entry is empty. Say, my line is `1,2,`. There is no position after the last `,` on which I can call `ParseValue`, so I need to add this extra column within the function call of the second-to-last column. Let me know if I managed to explain it, before resolving this point.